doc/source/cluster/kubernetes/k8s-ecosystem/yunikorn.md

```diff
@@ -0,0 +1,158 @@
+(kuberay-yunikorn)=
+# KubeRay integration with Apache YuniKorn
+
+[Apache YuniKorn](https://yunikorn.apache.org/) is a light-weight, universal resource scheduler for container orchestrator systems. It is created to achieve fine-grained resource sharing for various workloads efficiently on a large scale, multi-tenant, and cloud-native environment. YuniKorn brings a unified, cross-platform, scheduling experience for mixed workloads that consist of stateless batch workloads and stateful services.
+
+KubeRay's Apache YuniKorn integration enables more efficient scheduling of Ray pods in multi-tenant Kubernetes environments.
+
```

Comment:
Can you add a section `Prerequisites` between "KubeRay integration with Apache YuniKorn" and "Setup"? The section should mention which KubeRay version should be used (v1.2.2 or newer) and the API status (alpha for KubeRay v1.2.2). See https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/rayservice-quick-start.html as an example.